go/vt/servenv/run.go

```diff
@@ -40,13 +45,16 @@ func Run(port int) {
 	serveGRPC()
 	serveSocketFile()
 
-	l, err := proc.Listen(fmt.Sprintf("%v", port))
+	l, err := net.Listen("tcp", fmt.Sprintf(":%v", port))
 	if err != nil {
 		log.Exit(err)
 	}
 	go http.Serve(l, nil)
 
-	proc.Wait()
+	ExitChan := make(chan os.Signal, 1)
+	signal.Notify(ExitChan, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGINT)
```

Comment:
Not sure why we need to handle `SIGUSR1`. I see no references to this anywhere, but I carried it forward from the `proc` package.

Reply:
`SIGUSR1` was used as a custom signal by proc. So, we can drop it now.

Reply:
I've deleted it.